catalog.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Магазин мороженого "GllacyShop"</title>
+  </head>
+  <body>
+   <header>
+    <!--<p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p> -->
+     <nav>
+       <a href="index.html"><img src="" width="154" height="64" alt=""></a> <!--логотип магазина-->
+         <ul>
+            <li><a href="catalog.html">Каталог</a></li>
+            <li><a href="blank.html">Доставка и оплата</a></li>
+            <li><a href="blank.html">О компании</a></li>
+         </ul>
+         <ul>
+              <li><a href="blank.html">Поиск</a></li>
+              <li><a href="blank.html">Вход</a></li>
+              <li><a href="blank.html">Корзина</a></li>
+         </ul>
+         <ul>
+           <li><a href="index.html">Главная</a></li>
+           <li><a href="catalog.html">Каталог</a></li>
+           <li>Сливочное</li>
+         </ul>
+         <p>с 10 до 20, ежедневно</p>
+         <p><a href="tel:+78124502525">8 812 450-25-25</a></p>
+     </nav>
+   </header>
+<main>
+  <h1>Сливочное мороженое</h1>
+  <section>
+    <h2>Фильтр товаров</h2>
+    <form class="" action="http://echo.htmlacademy.html" method="post">
```

Comment:
В данном случае правильно использовать в форме метод `get`. Это фильтр и значение полей формы будут переданы в URL как get-параметры:
```suggestion
    <form class="" action="http://echo.htmlacademy.html" method="get">
```